### ISSUE TITLE
Unfortunately this service is unavailable if you are using a vpn

### ISSUE THREAD

**Issue by malex2005** [Seji64/SniDust#7]
Hi
I'm trying to reach Setanta sports Moldova with Snidust. Snidust works well and IP reported back is my VPS in Moldova. The DNS server is also in Moldova.
So I went on and added all the domains called (which gave me a nice long list see attached) but the only reply i get from the site is "Unfortunately this service is unavailable if you are using a vpn" 
The things is Setanta works well with a DNS proxy from one these commercial ones (tested with a free trial). I switched VPS to the same hosting company but no luck.

What have i missed or what could tweeked to stay under the radar? Has anyone had any issue like this with another service?

Alex

**Comment by malex2005** [comment]
Hi thanks for the update. I tested but same issue. 
What I noticed is when I do a test on ipleak[.]net (after adding the site in mydomains, and hiding my vps ip on the screenshot ) I only get 1 DNS server hit despite a high number of tests. If I use controlD for example I have multiple hits and no "VPN" detected . The location of the DNS servers seems to have no importance.

![image](https://user-images.githubusercontent.com/18400668/203406696-437a9338-11f5-483e-9b36-3bc14892c5dd.png)
